frontend/src/entry-forms/Transaction.svelte

```diff
@@ -91,6 +91,17 @@
       entry.postings = entry.postings;
     }
   }
+
+  function addPostingIfUnbalanced() {
+    const sum = entry.postings.reduce((agg: number, cur: Posting) => +cur.amount + agg, 0);
```

Comment:
This does not even work for simple amounts `+"20 EUR"` is `NaN. This needs to do some more work to actually get the "amount" from the posting in a way so that i can be summed.

Please also add tests for this logic.